_data/sidebar_doc.yml

```diff
@@ -366,6 +366,13 @@ entries:
 
         - title: Information Schema
           url: /information-schema.html
+          thirdlevel:
```

Comment:
For consistency with the rest of the file, would you mind formatting this as follows?

```
thirdlevel:
  - title: Porting Applications
    thirdlevelitems:

      - title: From PostgreSQL
        url: /porting-postgres.html
```